standalone/inc/pup/PUPPlaylist.cpp

```diff
@@ -106,8 +106,8 @@ PUPPlaylist* PUPPlaylist::CreateFromCSV(const string& line)
    }
 
    if (!hasFiles) {
-      PLOGD.printf("Playlist folder is empty");
-      return nullptr;
```

Comment:
Empty playlists were skipped, this then later caused warnings if they are referenced from triggers. We could later add this to a pup pack audit.